### PR TITLE
adds respawning for crashed car

### PR DESCRIPTION
**Pull request by DominikGuzei**

### CODE REVIEW

**Comment by stravid** [comment]
Looks good :+1:

File: app/assets/javascripts/game/controllers/car_controller.js.coffee [hunk 1]
```diff
@@ -29,13 +29,7 @@ Vector = helpers.math.Vector
...
     if @speed < 0 then @speed = 0
 
-  drive: ->
-    unless @crashing
-      @_driveOnPath()
-
-    else
-      @_calculateCrashingDirection()
-      @_updateCarPosition()
+  drive: -> unless @crashing then @_driveOnPath() else @_crash()
```
Comment (tombu) [inline app/assets/javascripts/game/controllers/car_controller.js.coffee:32]: I would prefer moving 'unless ... ' into the nex line

Comment (bmedicke) [inline app/assets/javascripts/game/controllers/car_controller.js.coffee:32]: I like it!

Comment (stravid) [inline app/assets/javascripts/game/controllers/car_controller.js.coffee:32]: I disagree :tea:

**Pull request merged**